sources/web/datalab/polymer/modules/GapiManager.ts

```diff
@@ -62,46 +79,58 @@ class GapiManager {
     const options = {
       prompt: promptFlags,
     };
-    return gapi.auth2.getAuthInstance().signIn(options);
+    return this.loadGapi()
+      .then(() => gapi.auth2.getAuthInstance().signIn(options));
   }
 
   /**
    * Signs the user out using gapi.
    */
    public static signOut(): Promise<void> {
-    return gapi.auth2.getAuthInstance().signOut();
+    return this.loadGapi()
+      .then(() => gapi.auth2.getAuthInstance().signOut());
   }
 
   /** Returns the signed-in user's email address. */
```

Comment:
Probably should reword this description, as it no longer actually returns the email address.